content/source/docs/enterprise-beta/api/team-members.html.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: enterprise2
+page_title: "Team Membership - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-team-members"
+---
+
+# Team Membership API
+The Team Membership API is used to add or remove users from a teams. The [Team API](/docs/enterprise-beta/api/teams.html) is used to create or destroy teams.
+
+-> **Note**: These API Endpoints are not yet available on `https://atlas.hashicorp.com/api/v2/` and may be subject to change before they are made available.
+
+## Add a User to Team
+This method adds multiple users to a Team. Both Users and Teams must already exist.
+
+| Method | Path           |
+| :----- | :------------- |
+| POST | /teams/:team_id/relationships/users |
+
+### Parameters
```

Comment:
you can add or delete multiple users from a team by including mutliple _user-identifer_ objects in the _data_ array.